tools/run_parallel.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Uncommenting the next line may be useful when debugging this script.
+#set -x
```

Comment:
Can we remove these 2 comment lines?

Reply:
Hi @lukech , I've now removed this code.